dash-renderer/src/reducers/callbacks.ts

```diff
@@ -136,9 +136,9 @@ export default (
                 return s;
             } else if (a.type === CallbackAggregateActionType.AddCompleted) {
                 return mutateCompleted(s, a);
-            } else {
-                return mutateCallbacks(s, a);
             }
+
+            return mutateCallbacks(s, a);
```

Comment:
Funny that these new rules prohibit `if(...) {return a} else { return b}` but not the same with `else if` 🤷